src/linker/cli.py

```diff
@@ -82,23 +82,28 @@ def run(
     """Run a pipeline from the command line."""
     configure_logging_to_terminal(verbose)
     logger.info("Running pipeline")
-    pipeline_specification = Path(pipeline_specification)
-    input_data = Path(input_data)
     results_dir = create_results_directory(output_dir, timestamp)
     logger.info(f"Results directory: {str(results_dir)}")
-    # TODO [MIC-4493]: Add configuration validation
+
+    pipeline_specification = Path(pipeline_specification).resolve()
```

Comment:
Why do we need these resolves?

Reply:
Oh good point - we already have them resolved from click's `resolve_path = True` option